emscripten/c_wrapper.cpp

```diff
@@ -20,6 +20,7 @@ extern "C" {
 /** declarations */
 
 void *vrvToolkit_constructor();
+void *vrvToolkit_constructorResroucePath(const char * resourcePath); //alternate constructor which allows user to set custom data path
```

Comment:
can't it be named `vrvToolkit_constructor(const char * resourcePath)` ?

Reply:
AFAIK, when compiling for C we can't have overloaded functions because of the name mangling. I tried compiling with the same function name, and both the emscripten and regular builds complain about a duplicate function definition. I don't know if there is a way around this without getting into complex macros and other preprocessing. Also, just committed a fix to the function name typo (fixed spelling Resrouce -> Resource).